lib/launcher.js

```diff
@@ -153,6 +154,26 @@ var init = function(configFile, additionalConfig) {
       }
     });
   }).then(function() {
+    // 3) If we're in `elementExplorer` mode, run only that.
+    if (config.elementExplorer) {
+      if (config.multiCapabilities.length != 1) {
+        throw new Error('Must specify only 1 browser while using elementExplorer');
```

Comment:
We should probably throw this warning if someone does something silly like manually set the framework to 'repl' (or 'explorer') but specifies multiCapabilities.